README.md

```diff
@@ -0,0 +1,84 @@
+# LBRY Block Explorer
+
+A simple PHP block explorer for browsing transactions and claims on the [LBRY](https://lbry.io) blockchain. The explorer was developed using CakePHP which is a model-view-controller (MVC) PHP framework.
+
+## Installation
+There are some prerequisites that need to be installed before the explorer can be accessed.
+* Web server - Apache, caddy or nginx
+* lbrycrd with txindex turned on
+* Python claims decoder (https://github.com/cryptodevorg/lbry-decoder)
+* MariaDB 10.2 or higher
+* Redis Server (optional, only required for the CakePHP redis cache engine, or to run `forevermempool`)
+* PHP 7.2 or higher
+  * php-fpm
+  * igbinary extension (https://github.com/igbinary/igbinary)
+  * phpredis extension (https://github.com/phpredis/phpredis)
+* composer (PHP package manager)
+
+### Installation steps
+* Clone the Github repository. `git clone https://github.com/lbryio/block-explorer`
+* Create a MariaDB database using the DDL found in `block-explorer/sql/lbryexplorer.ddl.sql`
+* Change the working directory to the cloned directory and run composer.
+```
+cd block-explorer
+composer update
+```
+* Create the directories, `tmp` and `logs` in the `block-explorer` folder if they have not been created yet, and make sure that they are writable by the web server.
+* Copy `config/app.default.php` to `config/app.php`. Edit the database connection values to correspond to your environment.
+* Copy `config/lbry.default.php` to `config/lbry.php`. Update the values for LBRY RPC URL and the Redis URL to correspond to your environment.
+* Configure your web server with the host root folder set to `<path to>/block-explorer/webroot` where `<path to>` is the absolute path to the configuraiton. Here is a sample nginx configuration. You can make changes to this configuration to correspond to your environment.
+```
+server {
+    listen        80;
+    server_name   my.explorer.com;
+
+    root /var/www/block-explorer/webroot;
+    index index.php;
+
+    location / {
+        try_files $uri $uri/ /index.php?$args;
+    }
+
+    # pass the PHP scripts to FastCGI server listening on the php-fpm socket
+    location ~ \.php$ {
+        try_files $uri =404;
+        include /etc/nginx/fastcgi_params;
+        fastcgi_pass 127.0.0.1:9000;
+        fastcgi_index index.php;
+        fastcgi_ignore_client_abort on;
+        fastcgi_param PHP_AUTH_USER $remote_user;
+        fastcgi_param PHP_AUTH_PW $http_authorization;
+        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+    }
+}
+```
+* Restart your web server.
+
+### Cron jobs
+There are a few scripts which can be set up as cron jobs or scheduled tasks.
+#### blocks.sh
```

Comment:
I think there should be newlines between paragraphs and the next header (throughout this section).